core/root.go

```diff
@@ -0,0 +1,59 @@
+package core
+
+import (
+	"fmt"
+	"github.com/zhanggbj/kperf/pkg/command/service"
+	"github.com/zhanggbj/kperf/pkg/command/version"
+	"os"
+
+	homedir "github.com/mitchellh/go-homedir"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"github.com/zhanggbj/kperf/pkg"
+)
+
+var cfgFile string
+
+// rootCmd represents the base command when called without any subcommands
+
+func NewPerfCommand(params ...pkg.PerfParams) *cobra.Command {
+	p := &pkg.PerfParams{}
+	p.Initialize()
+
+	rootCmd := &cobra.Command{
+		Use:   "kperf",
+		Short: "A CLI of to help with Knative performance test",
+		Long:  `A CLI of to help with Knative performance test.`,
+	}
+	cobra.OnInitialize(initConfig)
+	rootCmd.PersistentFlags().StringVar(&cfgFile, "config", "", "config file (default is $HOME/.config/kubectl)")
```

Comment:
The description here for defaulting is not the same as the actual logic, which one is correct?

Reply:
This is not used. Removing it.